src/freenas/etc/ix.rc.d/ix-sed

```diff
@@ -11,7 +11,13 @@
 
 unlock_drives()
 {
-	LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/midclt call disk.sed_unlock_all > /dev/null
+	# carp event script in the failover plugin has to call this
+	# since running this method on both controllers at the same
+	# time (on HA systems) can cause, ultimately, the disks to
+	# not get unlocked
+	if [ "$(ha_licensed)" = "false" ]; then
```

Comment:
will ha_license even be available here?

Reply:
I'd assume so since `ha_licensed` just calls the `hadetect` standalone script which exists on `/usr/local/bin` and we're calling `jq` which also exists on `/usr/local/bin`

Reply:
Oh right, I confused that with the shell script function, nevermind me

Reply:
Do I need to `. /etc/rc.freenas` in this rc script or will `. /etc/rc.subr` somehow source that function?